app/src/components/blue-subtitle.tsx

```diff
@@ -0,0 +1,15 @@
+import { TFunction } from "i18next";
```

Comment:
Use type imports if it's only used for type validation to reduce total bundle size.